code/extensions/ContentReviewDefaultSettings.php

```diff
@@ -123,8 +123,17 @@ public function updateCMSFields(FieldList $fields)
         $userField = ListboxField::create('OwnerUsers', _t('ContentReview.PAGEOWNERUSERS', 'Users'), $usersMap)
             ->setMultiple(true)
             ->setAttribute('data-placeholder', _t('ContentReview.ADDUSERS', 'Add users'))
-            ->setDescription(_t('ContentReview.OWNERUSERSDESCRIPTION', 'Page owners that are responsible for reviews'));
-
+					->setDescription(_t('ContentReview.OWNERUSERSDESCRIPTION', 'Page owners that are responsible for reviews'));
+
+		
+		if(Member::get()->count() > Config::inst()->get('SiteTree', 'content_review_gridfield_threshold')) {
+			$gridfieldconfig = GridFieldConfig_RelationEditor::create();
+			$gridfieldconfig->removeComponentsByType(new GridFieldAddNewButton());
+			$userField = GridField::create('OwnerUsers', _t("ContentReview.PAGEOWNERUSERS", "Users"),
+					$this->OwnerUsers()->Sort('FirstName'), $gridfieldconfig)
+					->setDescription(_t('ContentReview.OWNERUSERSDESCRIPTION', 'Page owners that are responsible for reviews'));
+		}
```

Comment:
This is starting to look better! I have two suggestions at this point:

* We should define the config prop on this extension as a `private static` and give it a PHPDoc block including `@config` annotation
* It might be a little nicer to add an `else` to this if statement which would add the `$userField` above rather than reassigning it